apps/condo/pages/news/[id]/index.tsx

```diff
@@ -358,7 +358,7 @@ const NewsItemCard: React.FC = () => {
                                                 messageType='secondary'
                                             />
                                         ),
-                                        isSent && (
+                                        isSending && (
```

Comment:
what about the case when NewsItem is sent? Judging by this logic we wont show the button :-)

Reply:
isSent || isSending?